cmd/collectors/zapi/plugins/qtree/qtree.go

```diff
@@ -421,11 +438,16 @@ func (q *Qtree) handlingQuotaMetrics(quotas []*node.Node, cluster string, quotaI
 						q.Logger.Debug().Msgf("(%s) failed to parse value (%s): %v", attribute, value, err)
 					} else {
 						*numMetrics++
-						q.Logger.Debug().Msgf("(%s) added value (%s)", attribute, value)
+						q.Logger.Trace().
+							Str("attribute", attribute).
+							Interface("value", value).
```

Comment:
lets not use Interface since it uses reflection, instead we can use Str("attrValue", attrValue)